Projects/ChangeTier/Sources/Service/OctopusImplementation/ChangeTierClientOctopus.swift

```diff
@@ -0,0 +1,268 @@
+import Foundation
+import hCore
+
+public class ChangeTierClientOctopus: ChangeTierClient {
+    public init() {}
+
+    public func getTier(contractId: String, tierSource: ChangeTierSource) async throws -> ChangeTierIntentModel {
+        /* TODO: REPLACE WITH REAL DATA */
```

Comment:
can we also try to add a little delay to see how its actually looking with the states before we get real endpoint.
`try await Task.sleep(nanoseconds: 2_000_000_000)`